CONTRIBUTING.md

```diff
@@ -1,80 +1,96 @@
 
-  - Push changes to your fork
-  - Create pull request
+ - `Hard fork bitcoin to a 2 MB blocksize limit.`
+ - `2 MB Test Net`
+ - `Consensus Rule changes - O(n^2) protections`
+ - `Activication / versionbits code`
 
-The title of the pull request should be prefixed by the component or area that the pull request affects. Examples:
+Emergencies and bug fixes are outside the scope of merge windows.
 
-    Consensus: Add new opcode for BIP-XXXX OP_CHECKAWESOMESIG
-    Net: Automatically create hidden service, listen on Tor
-    Qt: Add feed bump button
-    Trivial: fix typo
+## <a name="considerit"></a> Consider.it
+By voting on issues, Bitcoin Classic aims to create a collective deliberation among all those using this implementation. Every vote must be respected by 3 groups: users, miners, and developers. **The votes are interpreted as such:**
 
-If a pull request is specifically not to be considered for merging (yet) please prefix the title with [WIP] or use [Tasks Lists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) in the body of the pull request to indicate tasks are pending.
+- Each vote goes from -1 to 1, with 0 at neutral/undecided
+- In the Miner vote, each Miner's vote is multiplied by their hashpower
+- Only verified users have votes.
 
-The body of the pull request should contain enough description about what the patch does together with any justification/reasoning. You should include references to any discussions (for example other tickets or mailing list discussions).
+As voting is a continuous process, all voters are encouraged to vote for a pull request only if they believe it has been thoroughly tested, reviewed, and fitting of any other criteria the voter holds fit. If any vote is found to be subject of vote manipulation the vote will be considered void.
 
-At this stage one should expect comments and review from other contributors. You can add more commits to your pull request by committing them locally and pushing to your fork until you have satisfied all feedback. If your pull request is accepted for merging, you may be asked by a maintainer to squash and or rebase your commits before it will be merged. The length of time required for peer review is unpredictable and will vary from patch to patch.
+## <a name="codeofconduct"></a> Code of Conduct
 
+We want a productive, happy and agile community that can welcome new ideas in a complex field, improve every process every year, and foster collaboration between groups with very different needs, interests and skills. We gain strength from diversity, and actively seek participation from those who enhance it. This code of conduct exists to ensure that diverse groups collaborate to mutual advantage and enjoyment. We will challenge prejudice that could jeopardise the participation of any person in the project.
```

Comment:
This is very cool. Yes, we want to be productive, happy, and agile. Even though "agile" is such a buzzword.